src/intern.ts

```diff
@@ -20,39 +20,93 @@ export class intern {
     }
   }
 
-  static async findKnex(ent: any, q: any): Promise<any> {
+  static async findKnex(ent: any, q: any, knex: any): Promise<any> {
```

Comment:
`intern` should not be a class - it is never something we instantiate
just use a normal object with the knex functions are properties

Reply:
Ok! So much better that way!